src/applications/facility-locator/components/LocationHours.jsx

```diff
@@ -1,27 +1,61 @@
 import { values, every, capitalize } from 'lodash';
 import React, { Component } from 'react';
+import Raven from 'raven-js';
+import moment from 'moment';
 
 /**
  * VA Facility Known Operational Hours
  */
 export default class LocationHours extends Component {
-  colonizeTime(time) {
-    const found = time.match(/(\d?\d)(\d\d)(\w\w)/);
-    return `${found[1]}:${found[2]}${found[3]}`;
+  convertHour(hour, day) {
+    const hours = hour.split('-');
+    const isValid = hours.every(time => moment(time, 'hmmA').isValid());
+
+    if (!isValid) {
+      Raven.captureMessage('API data malformed', {
+        extra: {
+          data: this.props.location,
+          errorData: {
+            hours: { [day]: hour },
+          },
+        },
+      });
+
+      return '';
+    }
+
+    return hours.map(time => moment(time, 'hmmA').format('h:mmA')).join(' - ');
   }
 
-  renderNotes(notes) {
-    if (notes) {
-      return (
-        <div className="row">
-          <div className="small-12 columns">
-            <p>Notes: {notes}</p>
-          </div>
-        </div>
-      );
+  convertHours(hours) {
+    return Object.keys(hours).reduce((accum, key) => {
+      if (hours[key] === '-') {
+        return { ...accum, [key]: 'Closed' };
+      }
+
+      if (!this.convertHour(hours[key], key)) {
+        return { ...accum };
+      }
+
+      return { ...accum, [key]: this.convertHour(hours[key]) };
```

Comment:
Missing the second argument here, but see above comment about maybe removing that.